src/main/java/no/unit/nva/model/instancetypes/Monograph.java

```diff
@@ -0,0 +1,55 @@
+package no.unit.nva.model.instancetypes;
+
+import no.unit.nva.model.pages.MonographPages;
+import nva.commons.utils.JacocoGenerated;
+
+import java.util.Objects;
+
+public class Monograph implements PublicationInstance<MonographPages> {
+
+    private MonographPages pages;
+    protected boolean peerReviewed;
+
+    protected Monograph() {
+    }
+
+    protected Monograph(MonographPages pages, boolean peerReviewed) {
+        this.pages = pages;
+        this.peerReviewed = peerReviewed;
+    }
+
+    @Override
+    public MonographPages getPages() {
+        return pages;
+    }
+
+    @Override
+    public void setPages(MonographPages pages) {
+        this.pages = pages;
+    }
+
+    @Override
+    public boolean isPeerReviewed() {
+        return peerReviewed;
+    }
+
+    @JacocoGenerated
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof Monograph)) {
+            return false;
+        }
+        Monograph monograph = (Monograph) o;
+        return peerReviewed == monograph.peerReviewed
+                && Objects.equals(getPages(), monograph.getPages());
```

Comment:
here we use getters, and on the above line the field. I think we should adopt one unique strategy, and that is that all fields are private and if we need to access a field outside a class we use the accessors.